profiles/common/modules/features/cce_basic_config/cce_basic_config.module

```diff
@@ -358,3 +358,191 @@ function cce_basic_config_preprocess_container(&$variables) {
+        'link_path' => 'https://ec.europa.eu/info/cookies',
+        'menu_name' => 'menu-nexteuropa-service-links',
+        'weight' => -48,
+        'expanded' => 0,
+      ),
+      array(
+        'link_title' => 'Legal notice',
+        'link_path' => 'https://ec.europa.eu/info/legal-notice',
+        'menu_name' => 'menu-nexteuropa-service-links',
+        'weight' => -34,
+        'expanded' => 0,
+      ),
+      array(
+        'link_title' => 'Contact',
+        'link_path' => 'https://ec.europa.eu/info/contact',
+        'menu_name' => 'menu-nexteuropa-service-links',
+        'weight' => -21,
+        'expanded' => 0,
+      ),
+    ),
+    array(
+      array(
+        'link_title' => 'Facebook',
+        'link_path' => 'https://www.facebook.com/EuropeanCommission',
+        'menu_name' => 'menu-nexteuropa-social-media',
+        'weight' => 0,
+        'expanded' => 0,
+      ),
+      array(
+        'link_title' => 'Twitter',
+        'link_path' => 'https://twitter.com/eu_commission',
+        'menu_name' => 'menu-nexteuropa-social-media',
+        'weight' => 1,
+        'expanded' => 0,
+      ),
+      array(
+        'link_title' => 'Other social media',
+        'link_path' => 'http://ec.europa.eu/dgs/communication/services/journalist/social-media_en.htm',
+        'menu_name' => 'menu-nexteuropa-social-media',
+        'weight' => 2,
+        'expanded' => 0,
+      ),
+    ),
+  );
+
+  // Save menu group into menu_custom table.
+  foreach ($dt_menus as $menu) {
+    // Look the table first if the data does exist.
+    $exists = db_query("SELECT title FROM {menu_custom} WHERE menu_name=:menu_name", array(':menu_name' => $menu['menu_name']))->fetchField();
+    // Save the record if the data does not exist.
+    if (!$exists) {
+      menu_save($menu);
+    }
+  }
+
+  foreach ($dt_links as $layer1) {
+    foreach ($layer1 as $link) {
+      // Build an array of menu link.
+      $item = array(
+        'link_path' => $link['link_path'],
+        'link_title' => $link['link_title'],
+        'menu_name' => $link['menu_name'],
+        'weight' => $link['weight'],
+        'expanded' => $link['expanded'],
+      );
+      // Look the table first if the data does exist.
+      $exists = db_query("SELECT mlid from {menu_links} WHERE link_title=:link_title AND link_path=:link_path", array(':link_title' => $link['link_title'], ':link_path' => $link['link_path']))->fetchField();
+      // Save the record if the data does not exist.
+      if (!$exists) {
+        menu_link_save($item);
+      }
+    }
+  }
+
+  // Add block in site-wide context.
+  $context = module_invoke('context', 'load', 'site_wide');
+  foreach ($dt_menus as $menu_to_add) {
+    $context->reactions['block']['blocks'][$menu_to_add['menu_name']] = array(
+      'module' => 'menu',
+      'delta' => $menu_to_add['menu_name'],
+      'region' => $menu_to_add['region'],
+      'weight' => '0',
+    );
+  }
+
+  module_invoke('context', 'save', $context);
```

Comment:
Instead of this whole segment we can call the method multisite_drupal_toolbox_add_block_context